README.md

```diff
@@ -1,13 +1,33 @@
 # Mongo API Project
 
-Replace this readme with your own information about your project. 
+This project's goal is to look at how to use **Mongodb** to store data, and how to query that data from and API. This database is built using MongoDB, modelled and manipulated using **mongoose**. Then the **API** server is built with **Node on Express** and the aim is for my endpoints to follow **RESTful** standards. Some endpoints manipulate the data returned by making use of mongoose's specific **query comparison operators**. I'm once more using a Books-related dataset for this project 📚✨
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+## How I built it - What I learned
 
-## The problem
+- My first step was to get my database to work on **MongoDB Atlas**, since I knew my API server had to be deployed to Heroku. There were a lot of steps to follow, so it was a bit hard figuring out where to start, but once it was up and running, the rest of the project was smoother. Good point to remember while doing this setup was to add all needed Config Vars to Heroku so the database will be seeded and run as intended 😁
```

Comment:
Good reminder about the config vars. Was a bit tricky when deploying to herouku like you say 😊